### DISCUSSION TITLE
Where are the actions buttons?

### DISCUSSION THREAD

**Discussion by marcobarbosa** [sanity-io/sanity#3792] [Sanity Studio Support] [answered]
Hi,

When I create or update a document, I see that Sanity automatically saves it, but I never get to publish it. The action buttons are not there. 

<img width="157" alt="Screen Shot 2022-10-24 at 21 13 47" src="https://user-images.githubusercontent.com/61103/197653462-840738ca-8809-449e-9166-366f20ce668e.png">

The [documentation](https://www.sanity.io/docs/document-actions) describes how you could add "custom" actions, but I don't want anything custom. Just the standard actions: Publish, Delete, Duplicate, etc. Just like any of [your demos](https://next.sanity.build/desk/post;df0a7c4b-4500-4737-8ebd-5f9ff92d1bcc) or templates. [Next-sanity](https://github.com/sanity-io/next-sanity), for example, doesn't have any custom code, so what am I missing?

I'm using Sanity `^3.0.0-dev-preview.22` and `"next-sanity": "^0.8.4"`.

<img width="344" alt="Screen Shot 2022-10-24 at 21 17 02" src="https://user-images.githubusercontent.com/61103/197653726-2fc20646-c8cf-40bd-b301-d845e86678c6.png">

**Comment by mariuslundgard** [comment, marked as answer]
Hey @marcobarbosa – can you try upgrading to `"next-sanity": "^1.0.1"`?

**Reply by marcobarbosa** [reply]
Hey @mariuslundgard , thanks for the reply. Good call - I missed the latest tag.

BUT it's still the same. No buttons 🥲 Just auto save.

**Reply by stipsan** [reply]
Heya @marcobarbosa 👋 ! we meet again 😌  Do you have a public repo reproduction we could look at?

**Reply by marcobarbosa** [reply]
@stipsan It was a matter of time 🤓 I don't have a public repo, but could I invite you to a private one momentarily? It's in the early stage of setting up stuff.

**Reply by stipsan** [reply]
Sure fire me an invite and I'll debug 🙌

**Reply by stipsan** [reply]
Thanks for sharing your repo @marcobarbosa! @mariuslundgard it turned out to be another case of `@sanity/vision@^3.0.0-dev-preview.22` pulling in `3.0.0-v3-v2-pte.56` instead of `3.0.0-dev-preview.22` 😞 I'm not sure how we can prevent this from happening, beyond releasing `v3` on `latest` and stop using prerelease versions 😅

**Reply by mariuslundgard** [reply]
Oh snap – not easy to spot the little `^` 🧐

**Reply by marcobarbosa** [reply]
Glad you figured it out! Any short-term solution suggestions? I need to be able to publish documents asap 😬 

Anything I can change to pull in the right version?

**Reply by stipsan** [reply]
Yeah I made you a PR @marcobarbosa on your repo, the fix is to use exact versions and no semver. The equivalent of `npm i sanity@dev-preview @sanity/vision@dev-preview --save-exact` 🙌

**Reply by marcobarbosa** [reply]
Oh snap, amazing! Now we will stick to Sanity ❤️‍🔥
